packages/app/src/components/navigation/internalPages/index.tsx

```diff
@@ -0,0 +1,60 @@
+import classnames from 'classnames';
+import classNames from 'classnames';
+import React from 'react';
+import { Props as BaseProps } from '~/components';
+import BarsOutLineIcon from '~/components/icon/bars/outline';
+import HomeOutlineIcon from '~/components/icon/home/outline';
+import Link from '~/components/link';
+import { INTERNAL_PAGE_PATHS } from '~/constants';
+import { useI18n, useTranslation } from '~/hooks/i18n';
+import { Pathname, URL as _URL } from '~/types';
+
+const paths: {
+  to: Pathname | _URL;
+  label: string;
+  icon: JSX.Element;
+}[] = [
+  {
+    to: INTERNAL_PAGE_PATHS.ROOT,
+    label: 'internalPagePaths.root',
+    icon: <HomeOutlineIcon className="w=em" />,
+  },
+  {
+    to: INTERNAL_PAGE_PATHS.ENDPOINTS,
+    label: 'internalPagePaths.endpoints',
+    icon: <BarsOutLineIcon className="w=em" />,
+  },
+];
+
+type Props = BaseProps;
+const InternalPages: React.FC<Props> = ({ className = '', on }) => {
+  const { t } = useTranslation();
+
+  const { originalPath } = useI18n();
+  console.log(originalPath, on);
```

Comment:
nits: plz remove this line.

Reply:
[7c96903](https://github.com/cam-inc/viron/pull/674/commits/7c96903fb65162daa864db9bfb4126f98c23051b)